### ISSUE TITLE
Stuck in "The youtube-dl binary is being updated, please wait..."

### ISSUE THREAD

**Issue by i-need-to-tell-you-something** [jely2002/youtube-dl-gui#33]
**Describe the bug**
The app stays in the youtube-dl updating state forever. Logs in the end.

**To Reproduce**
1. Open the app (double click AppImage file or run from the terminal).
2. Observe the link input field say "The youtube-dl binary is being updated, please wait...".
3. Wait 15 min.
4. No change

**Expected behavior**
1. Open the app (double click AppImage file or run from the terminal).
2. Observe the link input field say "The youtube-dl binary is being updated, please wait...".
3. Wait 15 min.
4. Text changes to "input youtube link" or something and become clickable.

**Screenshots**
~~If applicable, add screenshots to help explain your problem.~~

**Additional info (please complete the following information):**
 - OS: Kubuntu 18.04 LTS
 - Application version 1.8.7

**Additional context**
Javascript console output:
<Details>

```
Settings loaded from file
caching.js:37 Cached directory exists
updater.js:54 Latest release: 2021.02.22
updater.js:93 Current version: 2020.11.17
updater.js:63 New version found! Updating...
updater.js:112 New youtube-dl version downloaded: 2021.02.22
updater.js:113 Writing version data...
updater.js:123 Adding chmod permissions
updater.js:126 chmod 0755 added
DevTools failed to load SourceMap: Could not parse content for file:///tmp/.mount_YouTubrpeObr/resources/app.asar/node_modules/human-signals/build/src/main.js.map: Unexpected end of JSON input
DevTools failed to load SourceMap: Could not parse content for file:///tmp/.mount_YouTubrpeObr/resources/app.asar/node_modules/human-signals/build/src/signals.js.map: Unexpected end of JSON input
DevTools failed to load SourceMap: Could not parse content for file:///tmp/.mount_YouTubrpeObr/resources/app.asar/node_modules/human-signals/build/src/core.js.map: Unexpected end of JSON input
DevTools failed to load SourceMap: Could not parse content for file:///tmp/.mount_YouTubrpeObr/resources/app.asar/node_modules/human-signals/build/src/realtime.js.map: Unexpected end of JSON input
DevTools failed to load SourceMap: Could not parse content for file:///tmp/.mount_YouTubrpeObr/resources/app.asar/web-resources/bootstrap.min.js.map: Unexpected end of JSON input
DevTools failed to load SourceMap: Could not parse content for file:///tmp/.mount_YouTubrpeObr/resources/app.asar/web-resources/bs-stepper.min.js.map: Unexpected end of JSON input
DevTools failed to load SourceMap: Could not parse content for file:///tmp/.mount_YouTubrpeObr/resources/app.asar/node_modules/uri-js/dist/es5/uri.all.js.map: Unexpected end of JSON input
DevTools failed to load SourceMap: Could not parse content for file:///tmp/.mount_YouTubrpeObr/resources/app.asar/node_modules/performance-now/lib/performance-now.js.map: Unexpected end of JSON input
```
</Details>
Terminal window output where I ran the AppImage:
<Details>

```
$ YouTube-Downloader-GUI-1.8.7.AppImage

(youtube-dl-gui.bin:13946): Gtk-WARNING **: 09:46:37.106: Theme parsing error: gtk.css:68:35: The style property GtkButton:child-displacement-x is deprecated and shouldn't be used anymore. It will be removed in a future version

(youtube-dl-gui.bin:13946): Gtk-WARNING **: 09:46:37.106: Theme parsing error: gtk.css:69:35: The style property GtkButton:child-displacement-y is deprecated and shouldn't be used anymore. It will be removed in a future version

(youtube-dl-gui.bin:13946): Gtk-WARNING **: 09:46:37.106: Theme parsing error: gtk.css:73:46: The style property GtkScrolledWindow:scrollbars-within-bevel is deprecated and shouldn't be used anymore. It will be removed in a future version
Checking for update
(node:13946) electron: The default of contextIsolation is deprecated and will be changing from false to true in a future release of Electron.  See https://github.com/electron/electron/issues/23506 for more information
details copied to home data
youtube-dl-darwin copied to home data
Update for version 1.8.7 is not available (latest version: 1.8.7, downgrade is disallowed).
checkForUpdatesAndNotify called, downloadPromise is null
ffmpeg copied to home data
```
</Details>

Anything I could test or try? I didn't try older versions, but I think I have had this issue before.

**Comment by jely2002** [comment]
Hi, i-need-to-tell-you-something,

I was not able to reproduce this issue using your linux distro and the mentioned version of the app.
However, I did find another bug that might make this bug happen more often. I have fixed that bug in the commit referenced above. This fix will be shipped in version 2.0.0.

In the mean time you can try the following:
- Remove the .youtube-dl-gui directory from your /home directory. This directory is hidden so you may have to enable "Show hidden files" in the settings of your file manager. After removing the .youtube-dl-gui folder try restarting the app.
- In my testing, just restarting the app helped as well.

Feel free to hit me up if you have any more questions.

**Comment by i-need-to-tell-you-something** [comment]
Yup, it worked fine after deleting that directory. Thanks for your help!
Also thanks a lot for making this! Looking forward to the re-write so I can recommend this as the default choice for everyone.
